lib/stripe/stripe_object.rb

```diff
@@ -208,9 +208,13 @@ def remove_accessors(keys)
         keys.each do |k|
           next if f.include?(k)
           next if @@permanent_attributes.include?(k)
-          k_eq = :"#{k}="
-          remove_method(k) if method_defined?(k)
-          remove_method(k_eq) if method_defined?(k_eq)
+
+          # Remove methods for the accessor's reader and writer.
+          [k, :"#{k}=", :"#{k}?"].each do |method_name|
```

Comment:
I fixed another small bug here in that that "question mark accessors" were added by `add_accessors` but not removed by `remove_accessors`. I doubt this contributed to any significant leakage though.